standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -705,12 +706,11 @@ public LockResponse lockMaterializationRebuild(String dbName, String tableName,
   public boolean heartbeatLockMaterializationRebuild(String dbName, String tableName, long txnId) throws MetaException {
     int result = jdbcResource.execute(
         "UPDATE \"MATERIALIZATION_REBUILD_LOCKS\"" +
-            " SET \"MRL_LAST_HEARTBEAT\" = " + Instant.now().toEpochMilli() +
-            " WHERE \"MRL_TXN_ID\" = " + txnId +
-            " AND \"MRL_DB_NAME\" = ?" +
-            " AND \"MRL_TBL_NAME\" = ?",
+            " SET \"MRL_LAST_HEARTBEAT\" = " + getEpochFn(jdbcResource.getDatabaseProduct()) +
```

Comment:
Instead of concatenating the timestamp, I would prefer to use it as parametrised value like txnId, dbName.

Reply:
I reverted that part to use a parameterzied query.

Reply:
HIVE-6965: Transaction manager should use RDBMS time instead of machine time. Determine the current time, using the RDBMS as a source of truth.

However, since it's only used for MV rebuild that's probably ok